simplenote_cli/sncli.py

```diff
@@ -1353,6 +1416,37 @@ def sncli_start(sync=sync, verbose=verbose, config=config):
         sn = sncli_start()
         sn.cli_note_markdown(key, 1 if args[0] == 'markdown' else 0)
 
+    # Tag API
+    elif args[0] == 'tag':
+
+        if not key:
+            usage()
+
+        if args[1] == 'get':
```

Comment:
should check the length of the args list before indexing - this and all `args[N]` below index error if the expected args not given. Eg `sncli -k somekey tag` or `sncli -k somekey tag set`.

Proposal: `tag` without `get` should default to displaying tags, other commands should should a helpful message if the tags list isn't given.

Reply:
:+1: to argument checking.

For the proposal, just to make sure I understand: You're saying `sncli -k <key> tag` should do the same thing as `... tag get`?

Reply:
Yes - I thought that might be a nice thing to default to. Otherwise it could simply print the usage info for all the tag functions. :)